addons/web/static/src/views/fields/properties/property_value.xml

```diff
@@ -115,7 +122,6 @@
                     <TagsList tags="propertyValue"/>
                     <Many2XAutocomplete
                         t-if="!props.readonly"
-                        id="''"
```

Comment:
Why is this one not receiving an ID?

Is this related to your commit message mentioning that it does not work for m2m ?
We could probably still pass an ID to it, for consistency and to potentially make it work "automatically" when the web team adds support for it.